### DISCUSSION TITLE
External audio routing on the server side

### DISCUSSION THREAD

**Discussion by Deftaudio** [apohl79/audiogridder#1153] [Ideas]
I'm considering using AG for managing substantial amount of outboard gear that I have in the same studio in other rooms. In such a way, I connect to the server and it routes the request to the external audio interface on the server. For sure I can use some VST plugin that allows initializing ASIO external driver, such as with Carla, I believe. But I'm wondering to have this functionality with AG possible? do you have other recommendations?

**Comment by Deftaudio** [comment]
Just checking if anyone has an idea here?

**Comment by apohl79** [comment]
Hey, we have a feature request for this functionality. Development is hopefully starting in the next few months. Thanks!

**Comment by Deftaudio** [comment]
Great news, thank you for sharing! Would be nice to see the list of features which are going to be developed.
